iocage_lib/ioc_start.py

```diff
@@ -745,13 +745,20 @@ def start_network_vnet_iface(self, nic, bridge, mtu, jid):
                 ],
                 stderr=su.STDOUT
             )
+            iocage_lib.ioc_common.checkoutput(
+                [
+                    'jexec', f'ioc-{self.uuid}', 'ifconfig', epair_b,
+                    'mtu', mtu
+                ],
+                stderr=su.STDOUT
+            )
 
-            if epair_b.decode() != jail_nic:
+            if epair_b != jail_nic:
                 # This occurs on default vnet0 ip4_addr's
                 iocage_lib.ioc_common.checkoutput(
                     [
                         "setfib", self.exec_fib, "jexec", f"ioc-{self.uuid}",
-                        "ifconfig", epair_b, "name", jail_nic, "mtu", mtu
+                        "ifconfig", epair_b, "name", jail_nic
```

Comment:
This command could include the mtu setting, so that the explicit jexec fork can be skipped in case `epair_b != jail_nic`.